plugin/build.gradle.kts

```diff
@@ -74,15 +76,15 @@ java {
 }
 
 tasks.withType<JavaCompile>().configureEach {
-    options.release.set(8)
+    options.release.set(11)
```

Comment:
So we can add AGP and get sourcesets defined by the plugin. AGP can't compile against java 8. Would need to rework this to probably use reflection for android projects if we don't want to bump version/include AGP.

Reply:
Indeed let's not bump the required version, as that affects all our users, and there's a lot of code using Java 8 that still needs to be converted. For that same reason we still support Gradle 4.10, as much as we're aware of benefits to future versions.